healthScore/templates/post_details.html

```diff
@@ -0,0 +1,223 @@
+        }
+        .post-container {
+            list-style-type: none;
+            padding: 0;
+            padding-left: 15%;
+            padding-right: 15%;
+        }
+        .comment-container {
+            list-style-type: none;
+        }
+        .comment {
+            width: 90%;
+            margin-top: 20px;
+            padding: 10px;
+            border: 1px solid #eee;
+            border-radius: 5px;
+        }
+        li {
+            margin-bottom: 20px;
+        }
+        .icon-bar {
+            display: flex;
+            justify-content: space-between;
+        }
+        .icon-space {
+            margin-right: 8px;
+        }
+        .btn-icon {
+            background: none;
+            border: none;
+            padding: 5px;
+            cursor: pointer;
+        }
+        .delete-btn {
+            color: rgb(250, 43, 43);
+        }
+    </style>
+</head>
+<body>
+{% include 'header.html' %}
+<br><br>
+<ul class="post-container">
+    {% if user.is_authenticated and post.user == user %}
+    <div class="icon-bar">
+        <div></div>
+        <div>
+            <a href="{% url "edit_post" post.id %}" class="btn btn-outline-primary btn-sm" role="button">Edit</a>
+            <!-- Delete Button -->
+            <button type="button" class="btn btn-outline-danger btn-sm" data-bs-toggle="modal" data-bs-target="#deleteConfirmModal">
+                Delete
+            </button>
+        </div>
+    </div>
+    {% endif %}
+    <li class="post">
+        <h2>{{ post.title }}</h3>
+        <small>{{ post.user.name }} • posted on {{ post.createdAt|date:"N j, Y, H:i:s" }} {% if post.updatedAt|date:"N j, Y, H:i:s" != post.createdAt|date:"N j, Y, H:i:s"%} • edited{% endif %}</small>
+        <hr>
+        <p>{{ post.description|linebreaks }}</p>
+        <div class="icon-bar">
+            <div>
+                <button type="button" class="btn-icon" aria-label="Like" >
+                    <i class="bi bi-heart-fill icon-space"></i> Like
+                </button>
+            </div>
+            <div>
+                <button type="button" class="btn-icon" aria-label="Collect">
+                    <i class="bi bi-bookmark-star-fill icon-space"></i> 
+                </button>
+                <button type="button" class="btn-icon" aria-label="Share">
+                    <i class="bi bi-share-fill"></i>
+                </button>
+            </div>
+          </div>
+    </li>
+    <hr>
+</ul>
+
+<div id="post-container">
+    <h2>All comments</h2>
+    <div>
+        <ul class="comment-container">
+            {% if comments %}
+            {% for comment in comments %}
+            <li class="comment">
+                <div class="icon-bar">
+                    <div>
+                        <h6>{{ comment.commenter }}</h6>
+                    </div>
+                    {% if user.is_authenticated %}
+                    {% if user == comment.commenter or user == comment.post.user %}
```

Comment:
Could we club these two conditions and write it like this:

if user.is_authenticated and (user == comment.commenter or user == comment.post.user) ?

Reply:
i tried but it didnt work, that's why

Reply:
Ohh. Alright then